src/aat/resources/features/F-066/S-182.td.json

```diff
@@ -0,0 +1,40 @@
+{
+  "title": "should retrieve trigger when the case and event exists",
+	"_guid_": "S-182",
+	"_extends_": "F-066_Test_Data_Base",
+
+	"specs": [
+		"an active profile in CCD",
+		"has the 200 return code"
+	],
+	"request": {
+    "pathVariables": {
+      "cid": "[[DYNAMIC]]"
```

Comment:
same as above

Reply:
Scenario 181 is removed as it was duplicate.